protocol/0056-REWA-rewards_overview.md

```diff
@@ -1,73 +1,120 @@
+
+Market activity (fee based) reward metrics are not stored in [LNL checkpoints](./0073-LIMN-limited_network_life.md) and are reset after a checkpoint restart.
 
-Reward metrics are scoped by market and measured per epoch hence are reset at the end of the epoch. 
 
-There will be the following fee metrics:
-1. Sum of maker fees paid (in a given market).
-1. Sum of maker fees received (in a given market).
-1. Sum of LP fees received (in a given market).
+### Market creation reward metrics
 
-There will be the following market creation metrics:
-1. Market total [trade value for fee purposes](0029-FEES-fees.md) since market creation multiplied by either `1` if the market creation reward for this market has never been paid or by `0` if the reward has already been paid.
+There will be a single creation reward metricand reward tyype. 
+This makes it possible to reward creation of markets achieving at least a minimum lifetime trading volume, as a proxy for identifying the creation of useful markets:
 
-Reward metrics are not stored in [LNL checkpoints](./0073-LIMN-limited_network_life.md). 
+Where:
 
-## Rewards accounts
+- there is a single eligible party for each market, which is the party that created the market by submitting the original new market governance proposal (**all other parties** have market creation metric = 0)
+- `cumulative volume` is defined as the cumulative total [trade value for fee purposes](0029-FEES-fees.md)
+- `rewards.marketCreationQuantumMultiple` is a network parameter described above
+-  `quantum` is an asset level field described in the [asset framework](0040-ASSF-asset_framework.md)
 
-Trading reward accounts are defined by the payment asset (the asset in which the reward is paid out), the market in scope, and the reward type (metric). I.e. there can be multiple rewards with the same type paid in different assets for the same market.
+The reward metric for the single *market creator* party is as follows:
 
-It must be possible for any party to run a one off [transfer](0057-TRAN-transfers.md) or create a [recurring transfer](0057-TRAN-transfers.md) to any of these reward accounts. 
-Note that the market settlement asset has nothing to do in particular with the asset used to pay out a reward for the market for any of the relevant trading rewards. 
+- **IF** `cumulative volume < rewards.marketCreationQuantumMultiple * quantum` **THEN** `market creation metric := 0`
+- **ELSE** `market creation metric := 1` (NB: this is 1 as market creation rewards are paid equally to all qualifying creators for *reaching* the volume threshold, not pro-rata based on cumulative volume)
+
+When the `market creation metric` for a party is `>0` and the reward account balance for a specific reward asset is also `>0` (i.e. when a creator is rewarded):
+
+- A flag is added for each of the recorded `funders` of the reward account paired with the market and reward asset.
```

Comment:
I think it needs to be clarified here. My understanding from the discussion on slack is that it should be possible to do the following:

transfer 1: funder: zohar, payout_asset: VEGA, markets_in_scope: [A,B,C]
transfer 2: funder: zohar, payout_asset: VEGA, markets_in_scope: [A,B]
transfer 3: funder: zohar, payout_asset: VEGA, markets_in_scope: []

all of the above will pay out A,B
the first and the third will pay out A,B,C 

Maybe should be clarified that the key for the flag is [funder, payout_asset, market_in_scope]. If a payout has been made for a combo, it shall never happen again, and is recorded in checkpoint as such.

Reply:
So the flag will be [funder, payout_asset, market_scope_set]? 

Just want to clarify as your last sentence used "market_in_scope" rather than "markets_in_scope" but I read the rest of the comment as suggesting there would be a separate flag for every scope.

Reply:
market**s**, as the examples read :)

Reply:
cool thanks